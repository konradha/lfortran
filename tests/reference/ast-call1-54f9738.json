{
    "basename": "ast-call1-54f9738",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form/call1.f",
    "infile_hash": "bc7d123659561bf2c5588a7485627e120019751a76c505c6bd9d1381",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-call1-54f9738.stdout",
    "stdout_hash": "197dde8679502298a3d200f6db5961682a7be7e1920e76ed38605b1e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}