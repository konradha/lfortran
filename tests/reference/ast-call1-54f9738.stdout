(TranslationUnit 
    [(Program 
        call1 
        () 
        [] 
        [] 
        [] 
        [(SubroutineCall 
            0 
            f 
            [] 
            [] 
            [] 
            ()
        )
        (SubroutineCall 
            0 
            f 
            [] 
            [] 
            [] 
            ()
        )
        (SubroutineCall 
            0 
            f_s 
            [] 
            [] 
            [] 
            ()
        )] 
        []
    )]
)
